specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/examples/ApiManagementListSku.json

```diff
@@ -0,0 +1,224 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
```

Comment:
>value [](start = 9, length = 5)

don't see "id", "name" and "type" as common properties in any of your items. resources should follow either "TrackedResource" or "ProxyResource" definition from [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json)

Reply:
This contract is the ARM contract for exposing SKUs for a provider, we just implemented the support